<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramDock</class>
 <widget class="QWidget" name="HistogramDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>956</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QComboBox" name="cbNormalization"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacerST">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>32</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="lBinRangesMin">
         <property name="text">
          <string>Minimum:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacerHT">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="hsName">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lOrientation">
         <property name="text">
          <string>Orientation:</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="lBinning">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Binning</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lXColumn">
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item row="26" column="0">
        <widget class="QLabel" name="lPlotRange">
         <property name="text">
          <string>Plot Range:</string>
         </property>
        </widget>
       </item>
       <item row="25" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="21" column="2">
        <widget class="NumberSpinBox" name="sbBinRangesMin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QComboBox" name="cbOrientation"/>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="lBinningMethod">
         <property name="text">
          <string>Method:</string>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <widget class="QLabel" name="lBinRangesMaxDateTime">
         <property name="text">
          <string>Maximum:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="TimedLineEdit" name="leName"/>
       </item>
       <item row="27" column="0">
        <spacer name="verticalSpacerGeneral">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lType">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="20" column="2">
        <widget class="QCheckBox" name="chkAutoBinRanges">
         <property name="text">
          <string>Auto</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="lBinWidth">
         <property name="text">
          <string>Bin Width:</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QLabel" name="lBinRangesMinDateTime">
         <property name="text">
          <string>Minimum:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lNormalization">
         <property name="text">
          <string>Normalization:</string>
         </property>
        </widget>
       </item>
       <item row="18" column="2">
        <widget class="NumberSpinBox" name="sbBinWidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacerST_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>32</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="20" column="0">
        <widget class="QLabel" name="lBinRanges">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Bin Ranges</string>
         </property>
        </widget>
       </item>
       <item row="17" column="2">
        <widget class="QSpinBox" name="sbBinCount">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="29" column="0">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>visible</string>
         </property>
        </widget>
       </item>
       <item row="22" column="2">
        <widget class="NumberSpinBox" name="sbBinRangesMax">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="16" column="2">
        <widget class="QComboBox" name="cbBinningMethod"/>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="lBinCount">
         <property name="text">
          <string>Bin Count:</string>
         </property>
        </widget>
       </item>
       <item row="24" column="2">
        <widget class="UTCDateTimeEdit" name="dteBinRangesMax">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="23" column="2">
        <widget class="UTCDateTimeEdit" name="dteBinRangesMin">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="ResizableTextEdit" name="teComment"/>
       </item>
       <item row="26" column="2">
        <widget class="QComboBox" name="cbPlotRanges"/>
       </item>
       <item row="7" column="2">
        <widget class="QComboBox" name="cbType"/>
       </item>
       <item row="22" column="0">
        <widget class="QLabel" name="lBinRangesMax">
         <property name="text">
          <string>Maximum:</string>
         </property>
        </widget>
       </item>
       <item row="28" column="0">
        <widget class="QCheckBox" name="chkLegendVisible">
         <property name="text">
          <string>Legend</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLine">
      <attribute name="title">
       <string>Shape</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>793</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSymbol">
      <attribute name="title">
       <string>Symbol</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabValues">
      <attribute name="title">
       <string>Values</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabAreaFilling">
      <attribute name="title">
       <string>Filling</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabErrorBars">
      <attribute name="title">
       <string>Error Bars</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Margins</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="0">
        <widget class="QLabel" name="lRugLength">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_13">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lRugEnabled">
         <property name="text">
          <string>Enabled:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lRugWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lRug">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Rug Plot</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lRugOffset">
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="chkRugEnabled">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="NumberSpinBox" name="sbRugLength">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="NumberSpinBox" name="sbRugWidth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="NumberSpinBox" name="sbRugOffset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>frontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>UTCDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>UTCDateTimeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>frontend/widgets/TimedLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>cbType</tabstop>
  <tabstop>cbOrientation</tabstop>
  <tabstop>cbBinningMethod</tabstop>
  <tabstop>sbBinCount</tabstop>
  <tabstop>chkAutoBinRanges</tabstop>
  <tabstop>chkVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
